plantower/plantower.py

```diff
@@ -134,3 +138,88 @@ def read(self, perform_flush=True):
                     return PlantowerReading(recv) # convert to reading object
             #If the character isn't what we are expecting loop until timeout
         raise PlantowerException("No message recieved")
+
+    def mode_change(self, mode=PMS_PASSIVE_MODE):
+        """
+            The default mode for the sensor is ACTIVE and whenever power OFF and ON (or sleep and wakeup)
+             the sensor starts in ACTIVE mode.
+            After mode changed, you need to wait some seconds for the sensor to get data again,
+            immediate reading sensor data gives you 0 values.
+        """
+
+        pms_cmd_changemode_passive = b'\x42\x4d\xe1\x00\x00\x01\x70'
+        pms_cmd_changemode_active = b'\x42\x4d\xe1\x00\x01\x01\x71'
+
+        if mode == PMS_PASSIVE_MODE:
+            self.serial.write(pms_cmd_changemode_passive)
+        else:
+            self.serial.write(pms_cmd_changemode_active)
+
+        self.serial.flush()
+        time.sleep(0.2)
+        return
+
+    def read_in_passive(self, perform_flush=True):
+        if perform_flush:
+            self.serial.reset_input_buffer()  #Flush any data in the buffer
+        pms_cmd_read_in_passive = b'\x42\x4d\xe2\x00\x00\x01\x71'
+        self.serial.write(pms_cmd_read_in_passive)
+        self.serial.flush()
```

Comment:
I suggest using `self.serial.reset_input_buffer()` instead of `self.serial.flush()`